chart/values.yaml

```
@@ -16,6 +16,19 @@ serviceAccount:
   # Annotations to add to the service account
   annotations: {}
 
+db:
+  # Setting this to true will create a postgres stateful set for config-db to connect to.
+  enabled: true
```

Not sure on the use case here? 

```suggestion
  enabled: false
```

---

The usecase is to provide a default database if apm-hub is installed separately as a logging solution. This will then be disabled in incident-commander-chart like we do with config-db and canary-checker. Wanted to keep all the setups homogeneous